utils/uiTypes/VotePlugin.ts

```diff
@@ -178,11 +198,63 @@ export class VotingClient {
       return { voterWeightPk, maxVoterWeightRecord: undefined }
     }
 
+    if (this.client instanceof HeliumVsrClient) {
```

Comment:
nit: not your fault but SURELY the way this stuff should be done, is that you would have a common interface for a `VotingClient`, and then each type of voting client implements that interface. NOT one giant class with a ton of if statements. 
Would it be too much work for you to adopt this approach for Helium? Extract whatever the API of this VotingClient is, make an interface for it, and then have `MegalithicOmniVotingClient` which is the current one, and `HeliumVotingClient`. And then whenever in the stores the voting client is set, it decides which to instantiate.

Reply:
Im not sure if I'll have the time to refactor our voting client to that, if I find it I can give it a go.